scRNA-seq/04-dimension_reduction_scRNA.Rmd

```diff
@@ -381,44 +398,49 @@ UMAP_plot_wrapper(nn_param = 5)
 
 #### Some 'big picture' thoughts to take from this experiment: 
 
-1. Analyses (such as UMAP) have various limitations for interpretability. 
+1. Analyses such as UMAP have various limitations for interpretability. 
 The coordinates of UMAP output for any given cell can change dramatically depending on parameters, and even run to run with the same parameters. 
-This probably means that you shouldn't rely too heavily on the exact values of UMAP's output. 
+This probably means that you shouldn't rely on the exact values of UMAP's output. 
 
-    - One particular limitation of UMAP and t-SNE is that while observed clusters have some meaning, the distance *between* clusters usually does not (nor does cluster density). 
+    - One particular limitation of UMAP (and t-SNE) is that while observed clusters have some meaning, the distance *between* clusters usually does not (nor does cluster density). 
     The fact that two clusters are near each other should NOT be interpreted to mean that they are more related to each other than to more distant clusters. 
     (There is some disagreement about whether UMAP distances have more meaning, but it is probably safer to assume they don't.)
-    
-2. Different analyses also have their strengths. 
-Using cell-type labeling, our experiment illustrated that UMAP does appear to give some biologically relevant output information for this dataset. 
 
-3. Playing with parameters so you can fine-tune them is a good way to give you more information about a particular analysis as well as the data itself. 
+
+2. Playing with parameters so you can fine-tune them is a good way to give you more information about a particular analysis as well as the data itself. 
+
+3. Where results are consistent, they are more likely to have meaning. 
+While we do not have labeled cell types in this case, there does seem to be some consistency of the overall patterns that we see (if not precise values), and this likely reflects biological information (or technical artifacts).
    
 In summary, if the results of an analysis can be completely changed by changing its parameters, you should be more cautious when it comes to the conclusions you draw from it as well as having good rationale for the parameters you choose. 
 
+
+
 ### t-SNE comparison
```

Comment:
Hm I could talk myself into removing this if we're concerned about time.

Reply:
It's just a little snack at the end. And I have plans to coming back to it later...